docs/platforms/react-native/sourcemaps/uploading/expo.mdx

```diff
@@ -47,6 +47,7 @@ npx sentry-expo-upload-sourcemaps dist
 
 - `dist` is the default output directory of `eas update`.
 - Uploaded source maps have no associated releases. This is expected as updates can apply to multiple releases.
+- Uploading of source maps for debug or simulator builds is disabled.
```

Comment:
These are notes for EAS Updates, meaning for situation when a user creates a JS bundle to distribute as ota update. 

This note is relevant for the previous section `[Automatic Upload for Release](https://docs.sentry.io/platforms/react-native/sourcemaps/uploading/expo/#automatic-upload-for-release)`

Reply:
Good catch @krystofwoldrich 🙇  I've moved the note to the correct section adding a short sentence for the why with 6d154f79f78b7773d441d927d35ff2e81e46ebb8.

> I think we should include the information also on the main source maps page. 

Also added a separate a bit more extended section on the main source maps page with e339fd8a050c0d61445cd692164855eb0591491b

Reply:
Thank you. I suggest removing the sim builds as release for sim will upload the source maps.

I also added @lizokm to review the docs style.

Reply:
Thank you @krystofwoldrich! Makes sense. I've committed your suggestions 🙇